test/minitest/test_minitest_mock.rb

```diff
@@ -227,17 +227,13 @@ def test_same_method_expects_are_verified_when_all_called
   end
 
   def test_same_method_expects_blow_up_when_not_all_called
-    mock = Minitest::Mock.new
-    mock.expect :foo, nil, [:bar]
-    mock.expect :foo, nil, [:baz]
-
-    mock.foo :bar
-
-    e = assert_raises(MockExpectationError) { mock.verify }
+    @mock = Minitest::Mock.new
+    @mock.expect :foo, nil, [:bar]
+    @mock.expect :foo, nil, [:baz]
```

Comment:
I saw this and the util method and felt like this was just forgotten, but then again, this does add an instance variable for no good reason, and worse than that will cause git churn for stylistic reasons. So I'll probably just leave it as is. Are you fine with that?